Race Trait Adaption

zumbak04 opened the pull request:
<!--
A basic changelog, goes to patch notes of the next version, so should be as short and informative as possible.
-->
## Changelog:
- Added all the race traits back (so far no effects, just placeholders).
- Every race trait has localisation and icon.

## How to test:
Enter the game, open the Character Finder, check character traits. Tell your opinion on icons and texts.

If you're a @Warcraft-GoA-Development-Team/english-watchers, please proofread the `.yml` files.

arithon commented on localization/english/wc_traits_l_english.yml:
```diff
@@ -0,0 +1,1027 @@
+﻿l_english:
+ trait_creature_human:0 "Human"
+ trait_creature_human_desc:0 "Many years ago, when the Titans left Azeroth, the old gods sent the Curse of the Flesh to the Titans' countless creations. The curse failed to bring them under the old gods' sway, but the vrykul - gloomy, cruel, and warlike giants of the north - were created to protect other servants of the Titans. Because of the curse, Vrykul women began to give birth to weak, ugly  ...
+ trait_creature_human_character_desc:0 "This character is a human. $trait_creature_human_desc$"
+ trait_creature_gnoll:0 "Gnoll"
+ trait_creature_gnoll_desc:0 "Gnolls are cunning, animal-like folk - a hybrid of man with hyena. A big hump, a long neck, and clumsy fingers do not stop them from fighting, and conflict itself is a way of life for this race. Gnolls are very violent creatures, even over the smallest dispute, they are liable to gnaw at each other's throats - and in a literal sense, as they devour all creatures, inc ...
+ trait_creature_gnoll_character_desc:0 "$trait_creature_gnoll_desc$"
+ trait_creature_murloc:0 "Murloc"
+ trait_creature_murloc_desc:0 "Murlocs are a strange combination of humanoid and fish, an ancient two-legged race of amphibians. They live along the coasts and within the depts of oceans, the coasts of lakes and rivers, as well as in underwater ruins. Murlocs are capable of breathing on land, although they rarely risk moving too far from water. Their hunched appearance is rather awkward, but don' ...
+ trait_creature_murloc_character_desc:0 "$trait_creature_murloc_desc$"
```
Why do we have "This character is a human", but not in the other trait descriptions?

zumbak04 replied:
> Why do we have "This character is a human", but not in the other trait descriptions?

`_character_desc` shows when the trait is on character. `_desc` shows when the trait is in an event option for example.

arithon replied:
Wouldn't it make more sense to put "This character is a ..." in all "trait_creature_..._character_desc" lines?

arithon replied:
Nvm, forget my previous question, I got it the wrong way.

arithon commented on localization/english/wc_traits_l_english.yml:
```diff
@@ -0,0 +1,1027 @@
+ trait_creature_wraith:0 "Spirit"
+ trait_creature_wraith_desc:0 "This character is an incorporeal spirt."
+ trait_creature_wraith_character_desc:0 "$trait_creature_wraith_desc$"
+ trait_lich_king:0 "Lich [Root.GetQueenKing]"
```
Does the scope still work?

zumbak04 replied:
> Does the scope still work?

Definitely not, I'll adjust it once the trait is here. So far, only races.

Nelpok reviewed (commented):
Seems all ancients are lacking on their traits

<details>

<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/51142337/94721177-de1f4380-032b-11eb-8ee7-309d5702568d.png)
![image](https://user-images.githubusercontent.com/51142337/94721193-e4adbb00-032b-11eb-8224-41cdecce9ed3.png)

</details>

Another thing that i noticed is the ammount of text there is on some traits for example humans have a massive wall of text of information, while others are too short, like night elves, or are directly laking information about them, like pandaren. I find this could be problematic because you also have to add the race modifiers (Opinion, Stat changes, etc) and it could be left...to big or just lacking in general

<details>

<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/51142337/94721307-14f55980-032c-11eb-820d-1a4fe0bc6796.png)
![image](https://user-images.githubusercontent.com/51142337/94721447-44a46180-032c-11eb-9f81-efcaad15febd.png)
![image](https://user-images.githubusercontent.com/51142337/94721470-4d953300-032c-11eb-8805-4ed503592864.png)

</details>

I also found those under creature that lack an image 

<details>

<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/51142337/94721693-9fd65400-032c-11eb-9212-d8d9dcb5d4b0.png)

</details>

All images for their races are GREAT

zumbak04 commented:
> Seems all ancients are lacking on their traits

Fixed in ff56bcb914c543e0632c24b5629cf2f321652c00.

> Another thing that i noticed is the ammount of text there is on some traits for example humans have a massive wall of text of information, while others are too short, like night elves, or are directly laking information about them, like pandaren. I find this could be problematic because you also have to add the race modifiers (Opinion, Stat changes, etc) and it could be left...to big or just lacking in general

Changed in 6603e2009e6e3228420d6ca84484b4d0550f5341, changed in e3e81ec7a074eac86bfc2258c592b144d3bd10af.

> I also found those under creature that lack an image

Fixed in e0acdc45feaafdc31e71f8d90c520e3e6d5458e1.

arithon reviewed (approved):
racial traits lgtm

MemeWizard commented on common/traits/wc_race_traits.txt:
```diff
@@ -0,0 +1,1488 @@
+		}
+	}
+}
+creature_furbolg = {
```
I suggest using less tech-savy furbolg as trait icon
Current one is holding a rifle on an icon

Aruzan replied:
Changed art to a more tribal looking Furbolg 460a2eaa0acff8acc5f38ca4e6188a3a7a3948f2

MemeWizard commented on common/traits/wc_race_traits.txt:
```diff
@@ -0,0 +1,1488 @@
+	}
+}
+
+creature_giant = {
```
ok but
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/98306360-baa78280-1fd4-11eb-9362-e3bc777c2615.png)

</details>

zumbak04 replied:
Changed in 63715f5fc6034c4596228752c75ab4708b7b08d9.

MemeWizard commented on common/traits/wc_race_traits.txt:
```diff
@@ -0,0 +1,1488 @@
+		}
+	}
+}
+creature_fel_orc = {
```
Dark background of this icon is kinda merges with dark ck3 interface background

Aruzan replied:
Changed art to a lighter Fel Orc picture to cause less blending with the background 460a2eaa0acff8acc5f38ca4e6188a3a7a3948f2

zumbak04 commented on localization/english/wc_traits_l_english.yml:
```diff
@@ -0,0 +1,1033 @@
+ trait_in_twisting_nether_desc:0 "Demons who are killed within the mortal world are sent to the Twisting Nether, a plane for immortal beings. Demons are trapped within stasis before they are summoned into the mortal world. There is no hierarchy or name for demons in the Twisting Nether - they only go by their merit, which may or may not be needed by their dominant fel masters."
+ trait_in_twisting_nether_character_desc:0 "$trait_in_twisting_nether_desc$"
+ trait_creature_lich:0 "Lich"
+ trait_creature_lich_desc:0 "Those that ascend into high tiers of necromantic magic sacrifice themselves for unspeakable unholy power. These people are known as liches, who take on the form of ethereal, skeletal beings which hover off of the ground, commanding thralls and undead armies. Liches can be independent, or servants to the Prince of Darkness."
```
Did another thing: the entire sentence didn't really make sense, so I deleted it in 908ca097e921302942998e97f94e1e357d9d5e05.
